posthog/temporal/batch_exports/bigquery_batch_export.py

```diff
@@ -215,57 +215,45 @@ async def insert_into_bigquery_activity(inputs: BigQueryInsertInputs) -> Records
         inputs.table_id,
     )
 
-    should_resume, details = await should_resume_from_activity_heartbeat(activity, BigQueryHeartbeatDetails, logger)
+    async with Heartbeater() as heartbeater:
+        await try_set_batch_export_run_to_running(run_id=inputs.run_id, logger=logger)
```

Comment:
TODO: Will move this `try_set_batch_export_run_to_running` to a background task.